packages/edit-site/src/components/sidebar-edit-mode/sidebar-card/style.scss

```diff
@@ -0,0 +1,34 @@
+.edit-site-sidebar-card {
```

Comment:
No strong feelings about it, but why aren't we using `Flex` or `HStack` and the like instead?

Reply:
Good point. I'm hesitant to change the implementation very much in this PR. I've tried to limit the changes to just moving the existing code as-is to a new reusable component.